src/httpserver/http_request.hpp

```diff
@@ -442,7 +444,7 @@ class http_request
         **/
         void set_arg(const std::string& key, const std::string& value)
         {
-            this->args[key] = value;
+            this->args[key] = value.substr(0,content_size_limit);
```

Comment:
It turns out to be enough to just check the size of `content`.  I added a function `content_too_large()`.

I also had to roll back pellucide's modification since it conflicted with checking the size.